src/services/typeorm.ts

```diff
@@ -83,12 +83,6 @@ export class TypeormWrapper {
     return typeorm;
   }
 
-  getMasterConnection(): Pool | undefined {
-    if (this.connection.driver instanceof PostgresDriver) {
-      return this.connection.driver.master as Pool;
-    }
-  }
-
```

Comment:
Nice catch. Does your editor tell you about unused methods?

Reply:
Yes it does. But I also do manual checks since it sometimes misfires.